signing_service/tests/tests.py

```diff
@@ -0,0 +1,4 @@
+
+
+def test_that_full_check_will_run_this_test():
+    assert True
```

Comment:
Please include at least rudimentary tests.

Reply:
Can it also be done in #634 ? Actually #634 changes it completely so all tests written here will have to be removed in #634.

Reply:
But then how do you know this code works? We don't know when #634 will pass the review. I'd really prefer each task to be self-contained. Maybe at least one tiny test? :)

Reply:
Ok, I will add some.

Reply:
:+1 done